### PR TITLE
tutorial.md: docker-server value for Docker Hub

### PR DESCRIPTION
**Pull request by didier-durand**
See comment in https://github.com/tektoncd/pipeline/issues/2616 

Most people are used to 'docker.io' as server for Docker Hub and not 'index.docker.io'. The tutorial doesn't work with 'docker.io', so defining the value clearly in case of widely-used Docker Hub may be useful.

Best,

Didier

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

# Release Notes

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -237,6 +237,7 @@ kubectl create secret docker-registry regcred \
                     --docker-password=<your-pword> \
                     --docker-email=<your-email>
 ```
+(If Docker Hub is used, then arg docker-server must be set to 'index.docker.io')
```
Comment (jerop) [inline docs/tutorial.md:240]: small nit, we tend to put these caveats after a `**Note:**` to ensure users don't miss them, as shown in this snippet from another part of this tutorial: 
https://github.com/tektoncd/pipeline/blob/b7e25054c9e07d44004f20e5618b4ffe354b4b8f/docs/tutorial.md#L14

**Review by jerop — COMMENTED** [review]
thanks @didier-durand!

/ok-to-test

**Comment by didier-durand** [comment]
@jerop : no interest in merging this typo fix? Didier

**Comment by chmouel** [comment]
@didier-durand i believe @jerop made a comment here https://github.com/tektoncd/pipeline/pull/3298#discussion_r496759397 and it probably can get merged quickly after that when this is addressed,

**Pull request closed without merging**